plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -79,6 +95,21 @@ const (
 // For example:
 // https://github.com/kubernetes/kubernetes/blob/36a218e/plugin/pkg/scheduler/factory/factory.go#L422
 
+// IMPORTANT: this list contains the ordering of the predicates, if you develop a new predicates
+// it is mandatory to add its name on this list.
+// otherwise it won't be processed, see generic_scheduler#podFitsOnNode()
+// the order is based on the restrictiveness & complexity of predicates
```

Comment:
It seems to be `Otherwise it` `The order` , `see generic_scheduler#podFitsOnNode().` Please check again to avoid any typo nits

Reply:
indeed, fixed